lib/mastodon/cli/search.rb

```diff
@@ -10,6 +10,7 @@ class Search < Base
       InstancesIndex,
       AccountsIndex,
       TagsIndex,
+      PublicStatusesIndex,
```

Comment:
Perhaps when using the `--only` CLI flag, we should automatically invoke both status indexes with the `statuses` option, otherwise it's inaccessible. But the CLI could be fixed/improved in a follow-up PR.